DEVELOPERS_GUIDE.md

```diff
@@ -204,28 +204,11 @@ export const environment = {
 
 ##### Client environment files
 
-We can repeat the process for the client. Create a file `environment.[NAME].ts` in `client/src/environments`.
-
-Also, add an entry in `angular-cli.json` accordingly:
-
-```
-...
-"environmentSource": "environments/environment.ts",
-"environments": {
-  "dev": "environments/environment.ts",
-  "personal": "environments/environment.personal.ts",
-  "offline": "environments/environment.offline.ts",
-  "staging": "environments/environment.staging.ts",
-  "production": "environments/environment.production.ts",
-  "test": "environments/environment.test.ts",
-  "e2e": "environments/environment.e2e.ts"
-}
-...
-```
+We can repeat the process for the client. Create a file `environment.personal.ts` in `client/src/environments`. This file is ignored by source control. Feel free to add other environment files by creating a `environment.[NAME].ts` file and adding an entry in [angular-cli.json](https://github.com/angular/angular-cli/wiki/build#build-targets-and-environment-files) accordingly.
```

Comment:
@PascalPrecht can you check this one again? If you like to have it differently, let me know and I'll change it.

Reply:
This LGTM